src/packagedcode/pypi.py

```diff
@@ -211,22 +230,34 @@ def parse_with_dparse(location):
         package_dependencies = []
         for df_dependency in df_dependencies:
             specs = df_dependency.specs
+            is_resolved = False
             requirement = None
             if specs:
                 requirement = str(specs)
+                if '==' in requirement:
```

Comment:
What about `===` and others quasi-equal in the latest specs?
Also since `df_dependencies.specs` is a list of Specifier packaging object https://github.com/pypa/packaging/blob/master/packaging/specifiers.py  why not use that instead of taking a `str()` on it?
You could check if the `.operator` matches a list of equality operators AND that there is only one specifier (which is the only valid case for `is_resolved` ) 
See https://github.com/pypa/packaging/blob/master/packaging/specifiers.py#L173